### PR TITLE
DT-1398 Use latest gradle plugin

### PR DESCRIPTION
**Pull request by mikehalmamoj**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 plugins {
-  id("uk.gov.justice.hmpps.gradle-spring-boot") version "1.1.2"
+  id("uk.gov.justice.hmpps.gradle-spring-boot") version "2.1.0"
   kotlin("plugin.spring") version "1.4.10"
```
Comment (andymarke) [inline build.gradle.kts:3]: Upgrade kotlin as well?

**Review by andymarke — APPROVED** [review]
LGTM

File: build.gradle.kts (hunk 1 above)
Comment (mikehalmamoj) [reply build.gradle.kts:3]: Thanks!

**Review by andymarke — APPROVED** [review]
LGTM

**Pull request merged**